pmlc/dialect/pxa/transforms/vectorize.cc

```diff
@@ -305,15 +305,16 @@ LogicalResult vectorizeOverOutputs(AffineParallelOp op, unsigned vectorWidth) {
   }
   auto reduce = dyn_cast<PxaReduceOp>(getPrevWriter(op.getResult(0)));
   if (!reduce) {
-    return op.emitRemark(
-        "vectorizeOverOutputs: Failed, missing previous PxaReduceOp");
+    return failure();
```

Comment:
We are hitting this case a LOT on ResNet50 which bulks up the logs quite a bit as we print the entire op IR when we do a remark.  Not sure if this is the right thing to do here, or not.